lib/openstax/aws/stack.rb

```diff
@@ -168,6 +168,8 @@ def create_change_set(options)
     def apply_change_set(params: {}, wait: false)
       logger.info("**** DRY RUN ****") if dry_run
 
+      @previous_parameters = deployed_parameters
```

Comment:
Do we need to persist the previous parameters or template somewhere?  If the process dies, these instance variables will be lost and we will no longer be able to revert.

Reply:
Hmm, that's a good question. I'm not sure if this should be done in this or a in a separate issue. I also don't know what would be the best way to achive this - some temp file?
cc @TomWoodward